api/config/module/chaos/experimentation/xds/v1/xds.proto

```diff
@@ -26,4 +26,12 @@ message Config {
 
   // The interval at which to send heartbeat responses for TTL'd resources.
   google.protobuf.Duration heartbeat_interval = 6;
+
+  message ECDSAllowList {
+    repeated string enabled_clusters = 1;
+  }
+
+  // Specifies which clusters ECDS is enabled for, causing faults to be communicated over ECDS instead of RTDS where
+  // supported. If unset, ECDS will be used for all clusters.
+  ECDSAllowList ecds_allow_list = 7;
```

Comment:
I might have suggested this originally but I think we'd want some kind of gating to prevent all configuration to suddenly start using ECDS. Maybe something like

```
message ECDSConfiguration {
  message AllowList {
    repeated string enabled_clusters = 1;
  }

  AllowList allow_list = 1;
}

EcdsConfiguration ecds = 1;
```

With `ecds: {}` meaning enable it for everything, and `ecds: { allow_list: { enabled_clusters: [...] } }` using a list of clusters? Alternatively we can oneof this with a bool always true param, but I'm not sure if that is much cleaner.

The tl;dr is that we don't want someone pulling in this change to switch everything to ECDS if they don't change their configuration..

Reply:
Having read the rest of the code we could also just document that this doesn't have a "allow all" mode at the moment and leave the API as is